pages/courses_page.py

```diff
@@ -151,3 +154,21 @@ def verify_footer_card_images(self, selector, verification_list):
                 print(image_name + ' not found!')
                 return False
         return True
+
+    def verify_href_response_code(self, element_list):
+        """
+        Explore a course and verify that all course cards are loaded with response code OK"
+        """
+        time.sleep(3)
+        images_list = self.get_element_list(element_list)
+        try:
+            for element in images_list:
+                value = element.get_attribute('href')
+                r = requests.get(value)
+                response_code = r.status_code
+                if response_code == 200:
```

Comment:
```
if r.status_code == 200
```
We can remove the extra variable at line 168 

also, try to avoid variables with single letter
```
r = 'something' #bad 
response_code = 'something' #good 
```